lxd/network/acl/driver_common.go

```diff
@@ -10,7 +10,7 @@ import (
 	"strings"
 	"sync"
 
-	"github.com/canonical/lxd/client"
+	lxd "github.com/canonical/lxd/client"
```

Comment:
This I think is unwanted change that VSCode automatically does IIRC.

Reply:
ah yes, I forgot this one....

Reply:
fixed